### DISCUSSION TITLE
Diacritics

### DISCUSSION THREAD

**Discussion by isodabas** [codingjoe/django-select2#50] [Q&A]
I've looked for diacritics support but i couldn't find it.

Does django-select2 suppport diacritics?

**Comment by codingjoe** [comment]
Hi @isodabas,

Thanks for reaching out. I am afraid though, I don't really know what you mean. Are you talking about letter agents or some kind of Python package I am not familiar with?

If the former, sure why not? If your keyboard allows you to enter characters with diacritics you should find results for those.
This package does support Unicode, at least it should.

However, we do not do stemming or anything similar. If you want `á` if you type `a` that is something your database should probably handle.

I hope this answer your question. If not, please feel free to expand your question.

Best,
Joe
